lib/active_model/serializer.rb

```diff
@@ -31,6 +31,7 @@ def self.inherited(base)
     def self.attributes(*attrs)
       attrs = attrs.first if attrs.first.class == Array
       @_attributes.concat attrs
+      @_attributes.uniq!
```

Comment:
Well, I can see 3 use cases:
* a user calls `attributes :title, :title`
* a user calls: 
```ruby
attributes :title
attributes :title
```
* a future feature to call `attributes` in a similar way the fragment cache

The first two are not really serious, but the last one might go unseen like the fragment cache.